src/cluster/ec2_pke.go

```diff
@@ -859,7 +859,8 @@ func (c *EC2ClusterPKE) GetBootstrapCommand(
 			"--kubernetes-api-server=%q "+
 			"--kubernetes-cluster-name=%q "+
 			"--kubernetes-master-mode=%q "+
-			"--kubernetes-advertise-address=0.0.0.0:6443",
+			"--kubernetes-advertise-address=0.0.0.0:6443 "+,
```

Comment:
**Required**: The comma **MUST** be removed from here because it is syntactically incorrect.

Reply:
thx